modules/opensimplex/doc_classes/OpenSimplexNoise.xml

```diff
@@ -30,9 +30,14 @@
 			<argument index="0" name="width" type="int">
 			</argument>
 			<argument index="1" name="height" type="int">
-			</argument>
+      </argument>
+      <argument index="2" name="noise_offset_x" type="int" default="0">
+      </argument>
+      <argument index="3" name="noise_offset_y" type="int" default="0">
+      </argument>
 			<description>
-				Generate a noise image in [constant Image.FORMAT_L8] format with the requested [code]width[/code] and [code]height[/code], based on the current noise parameters.
+				Generate a noise image in [constant Image.FORMAT_L8] format with the requested [code]width[/code] and [code]height[/code], based on the current noise parameters. Optionally specify [code]noise_offset_x[/code] and [code]noise_offset_y[/code] values which are used as the coordinates of the
+        top left corner of the generated noise.
```

Comment:
Don't wrap lines, as this creates new paragraphs in the editor help:

```suggestion
				Generate a noise image in [constant Image.FORMAT_L8] format with the requested [code]width[/code] and [code]height[/code], based on the current noise parameters. Optionally specify [code]noise_offset_x[/code] and [code]noise_offset_y[/code] values which are used as the coordinates of the top left corner of the generated noise.
```

Reply:
Fixed. Thanks for the feedback.